src/btreeInt.h

```diff
@@ -521,7 +521,7 @@ struct BtCursor {
   u8 curIntKey;             /* Value of apPage[0]->intKey */
   u16 ix;                   /* Current index for apPage[iPage] */
   u16 aiIdx[BTCURSOR_MAX_DEPTH-1];     /* Current index in apPage[i] */
-  struct KeyInfo *pKeyInfo;            /* Arg passed to comparison function */
+  _Alignas(16) struct KeyInfo *pKeyInfo;            /* Arg passed to comparison function */
```

Comment:
Always 16?  Not `SQLITE_DEFAULT_ALIGNMENT`?

Reply:
Probably not needed, I just added that for debugging (`assert(EIGHT_BYTE_ALIGNMENT(&pCur->pKeyInfo));`). Will revert.